tests/test_enterprise/api_client/test_discovery.py

```diff
@@ -416,7 +427,21 @@ def test_is_course_in_catalog(self, catalog_id, course_id, api_resp, expected, m
         self.api = CourseCatalogApiClient(self.user_mock)
         assert self.api.is_course_in_catalog(catalog_id, course_id) == expected
         discovery_client.catalogs.assert_called_once_with(catalog_id)
-        discovery_client.catalogs.return_value.contains.get.assert_called_once_with(course_run_id=course_id)
+        if is_run:
+            discovery_client.catalogs.return_value.contains.get.assert_called_once_with(course_run_id=course_id)
+        else:
+            discovery_client.catalogs.return_value.contains.get.assert_called_once_with(course_id=course_id)
+
+    @ddt.data(
+        (None, []),
+        ({}, []),
+        ({'courses': [{'key': 'first+key'}, {'key': 'second+key'}]}, ['first+key', 'second+key'])
+    )
+    @ddt.unpack
```

Comment:
BTW by 'unnest' I meant put them on separate lines -- sorry if that was confusing.